grommet/grommet
# Available boolean: now accessible from extended props

#### What does this PR do?
Aims to resolve #5496. Introduces `Grid.available` as a part of the `GridExtendedProps` to stop TypeScript from marking it as an issue when trying to access `Grid.available` in your own code.

#### Where should the reviewer start?
`Grid/index.d.ts`: to view the addition of the boolean to GridExtendedProps
`Grid/Grid.js`: to see where it is given a value

#### What testing has been done on this PR?
Manual testing in Grid's TypeScript story (`Grid/stories/typescript/App.tsx`) by trying to console log `Grid.available`.

#### How should this be manually tested?
1. Open Grid's TypeScript story (`Grid/stories/typescript/App.tsx`)
2. Try to log/access `Grid.available` and notice how TypeScript does not complain about the `available` property

#### Any background context you want to provide?

#### What are the relevant issues?
Issue #5496 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
